scripts/github/update-gh-pages.sh

```diff
@@ -161,11 +214,11 @@ echo "Committing changes..."
 git add .
 git commit $amend -m "$commit_msg"
 
-cd -
+popd > /dev/null
 
 echo "gh-pages branch successfully updated"
 
 if [ -n "$push_remote" ]; then
     echo "Pushing gh-pages to $push_remote"
-    git push ${amend+-f} "$push_remote" gh-pages
+    git push --force ${amend+-f} "$push_remote" gh-pages
```

Comment:
I'm not sure we want to `--force`. If the push fails (because of git history doesn't apply) we have some problem we better get notified and investigate the issue

Reply:
removing.... it was for devel forgot to rm